genesis/Cargo.toml

```diff
@@ -12,7 +12,7 @@ workspace = true
 
 [dependencies]
 iroha_crypto = { workspace = true }
-iroha_data_model = { workspace = true, features = ["http"] }
+iroha_data_model = { workspace = true, features = ["http", "transparent_api"] }
```

Comment:
```suggestion
iroha_data_model = { workspace = true, features = ["http"] }
```

If clients are expected to use this library, this is not ok

Reply:
I can move this function to `iroha_data_model`, do you think it is ok?
```rust
fn create_genesis_block(
    genesis_transaction: SignedTransaction,
    topology: Vec<PeerId>,
) -> SignedBlock {
```

Reply:
Maybe BlockBuilder can have build_genesis?

Reply:
It will be cyclic dependency if we use `BlockBuilder` in `iroha_genesis`. Added method `SignedBlock::new_genesis_block`

Reply:
not the best but fine